### PR TITLE
Detailed description for recursion in Fibonacci

### PR DESCRIPTION
**Pull request by race2infinity**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Review by thecodingaviator — DISMISSED** [review]
Fix merge conflict please

**Review by thecodingaviator — COMMENTED** [review]
Hi,

As there has been no response from your end for a while, we are closing this PR.

Feel free to request this PR to be reopened, if you still wish to make the changes you originally added.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding! ✈️

**Comment by kylelobo** [comment]
Hey, could you please reopen this PR?

**Comment by kylelobo** [comment]
Sorry to trouble you, but I tried resolving conflicts and I am still getting "Changes requested". Did I go wrong somewhere?

**Review by race2infinity — COMMENTED** [review]
Resolved conflicts and updated "More Information" links

File: guide/english/computer-science/dynamic-programming/index.md [hunk 1]
```diff
@@ -15,34 +15,37 @@ This is the tree to find F(5):
 
 ![Fibonacci serie's tree](https://cdn-media-1.freecodecamp.org/imgr/59Rpw.png)
 
-To compute F(5) it will need to compute many times the same F(i). Using recursion:
+To compute F(5), the recursive method will compute the same value multiple times
 
+For example:
+
+F(0) is called 3 times </br>
```
Comment (thecodingaviator) [inline guide/english/computer-science/dynamic-programming/index.md]: Please remove the line breaks. The way we display articles, they’ll get added automatically

Comment (kylelobo) [reply guide/english/computer-science/dynamic-programming/index.md]: Done!

**Review by thecodingaviator — APPROVED** [review]
LGTM

**Comment by thecodingaviator** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**